--- enchant-1.6.1/configure.ac.orig	2017-04-04 23:13:57.000000000 +0200
+++ enchant-1.6.1/configure.ac	2018-12-23 12:14:42.036264559 +0100
@@ -386,7 +386,7 @@
          ,)
        AC_CHECK_LIB(aspell,get_aspell_dict_info_list,ASPELL_CFLAGS="-DASPELL_0_50_0=1",)
 
-       ASPELL_CFLAGS+=" -DHAVE_PSPELL_H"
+       ASPELL_CFLAGS="$ASPELL_CFLAGS -DHAVE_PSPELL_H"
    elif test -f "$aspell_prefix/include/aspell.h"; then
        AC_MSG_RESULT([yes (aspell)])
 
@@ -511,9 +511,9 @@
     AC_ARG_ENABLE(applespell, AS_HELP_STRING([--disable-applespell],[enable the applespell backend @<:@default=no@:>@]), build_applespell="$enableval", build_applespell=no)
 
     if test "x$build_applespell" != "xno"; then
-        APPLESPELL_CFLAGS+=" -DXP_TARGET_COCOA -xobjective-c"
-        APPLESPELL_LIBS+=" -lobjc"
-        APPLESPELL_LDFLAGS+=" -framework Cocoa"
+        APPLESPELL_CFLAGS="$APPLESPELL_CFLAGS -DXP_TARGET_COCOA -xobjective-c"
+        APPLESPELL_LIBS="$APPLESPELL_LIBS -lobjc"
+        APPLESPELL_LDFLAGS="$APPLESPELL_LDFLAGS -framework Cocoa"
     fi
 
 fi
